### PR TITLE
Add PropTypes to all teacher components except for TeacherLessons

### PR DESCRIPTION
**Pull request by KyleWong2510**
## Pull request checklist
Please check if your PR fulfills the following requirements:
- [ ] Lint (`npm run lint`) has passed locally and any fixes were made for failures
## Pull request type
 
 
Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [x] Testing
- [ ] Other (please describe): 
## What is the current behavior?

- 
## What is the new behavior?

- Add proptypes to all teacher components
## Does this introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

File: src/CreateStudentForm/CreateStudentForm.js [hunk 1]
```diff
@@ -87,4 +88,9 @@ const mapDispatchToProps = (dispatch) =>
...
   )
 
-export default connect(null, mapDispatchToProps)(CreateStudentForm)
\ No newline at end of file
+export default connect(null, mapDispatchToProps)(CreateStudentForm)
+
+CreateStudentForm.propTypes = {
+  completeForm: PropTypes.func.isRequired,
+  addStudent: PropTypes.func.isRequired
+}
```
Comment (Melizzo) [inline src/CreateStudentForm/CreateStudentForm.js:96]: beautiful

File: src/TeacherRoster/TeacherRoster.js [hunk 2]
```diff
@@ -62,3 +63,7 @@ const mapStateToProps = ({ setStudents }) => ({
 })
 
 export default connect(mapStateToProps)(TeacherRoster)
+
+TeacherRoster.propTypes = {
+  students: PropTypes.array.isRequired
+}
```
Comment (Melizzo) [inline src/TeacherRoster/TeacherRoster.js:69]: Wow, you got alot done on this Kyle, thank you

**Review by Melizzo — APPROVED** [review]

File: src/LessonPreview/LessonPreview.js [hunk 3]
```diff
@@ -93,3 +94,12 @@ const mapDispatchToProps = (dispatch) =>
   );
 
 export default connect(null, mapDispatchToProps)(LessonPreview);
+
+LessonPreview.propTypes = {
```
Comment (Billwilke42) [inline src/LessonPreview/LessonPreview.js:98]: nice work on these

**Review by Billwilke42 — COMMENTED** [review]
Nice work!

**Review by Billwilke42 — APPROVED** [review]
Approved

**Pull request merged**